src/tests/test.R

```diff
@@ -12,4 +12,5 @@ source("src/tests/webDriverNavigation.R")
 
 tryCatch({
     testthat::test_dir("src/tests/testthat", stop_on_failure = TRUE, stop_on_warning = TRUE)
+    #testthat::test_file("src/tests/testthat/testMetadata.R")
```

Comment:
This line can go! Then go ahead and merge

Reply:
I was deliberately leaving that it, so that it's easy in the future to switch over to just testing a single file by commenting one line, uncommenting the other, and updating the path.